plugin/trino-druid/src/test/java/io/trino/plugin/druid/TestDruidTypeMapping.java

```diff
@@ -170,50 +180,76 @@ public void testVarchar()
+                .add(new TimestampCase("1970-01-01 00:00:00.000", "TIMESTAMP '1970-01-01 00:00:00.000'", id++))
                 // time gap in JVM zone
-                {"1970-01-01 00:13:42.000", "TIMESTAMP '1970-01-01 00:13:42.000'"},
-                {"2018-04-01 02:13:55.123", "TIMESTAMP '2018-04-01 02:13:55.123'"},
+                .add(new TimestampCase("1970-01-01 00:13:42.000", "TIMESTAMP '1970-01-01 00:13:42.000'", id++))
+                .add(new TimestampCase("2018-04-01 02:13:55.123", "TIMESTAMP '2018-04-01 02:13:55.123'", id++))
                 // time gap in Vilnius
-                {"2018-03-25 03:17:17.000", "TIMESTAMP '2018-03-25 03:17:17.000'"},
+                .add(new TimestampCase("2018-03-25 03:17:17.000", "TIMESTAMP '2018-03-25 03:17:17.000'", id++))
                 // time gap in Kathmandu
-                {"1986-01-01 00:13:07.000", "TIMESTAMP '1986-01-01 00:13:07.000'"},
+                .add(new TimestampCase("1986-01-01 00:13:07.000", "TIMESTAMP '1986-01-01 00:13:07.000'", id++))
                 // test arbitrary time for all supported precisions
-                {"1970-01-01 00:00:00", "TIMESTAMP '1970-01-01 00:00:00.000'"},
-                {"1970-01-01 00:00:00.1", "TIMESTAMP '1970-01-01 00:00:00.100'"},
-                {"1970-01-01 00:00:00.12", "TIMESTAMP '1970-01-01 00:00:00.120'"},
-                {"1970-01-01 00:00:00.123", "TIMESTAMP '1970-01-01 00:00:00.123'"},
-                {"1970-01-01 00:00:00.1239", "TIMESTAMP '1970-01-01 00:00:00.123'"},
-                {"1970-01-01 00:00:00.12399", "TIMESTAMP '1970-01-01 00:00:00.123'"},
-                {"1970-01-01 00:00:00.123999", "TIMESTAMP '1970-01-01 00:00:00.123'"},
-                {"1970-01-01 00:00:00.1239999", "TIMESTAMP '1970-01-01 00:00:00.123'"},
-                {"1970-01-01 00:00:00.12399999", "TIMESTAMP '1970-01-01 00:00:00.123'"},
-                {"1970-01-01 00:00:00.123999999", "TIMESTAMP '1970-01-01 00:00:00.123'"},
+                .add(new TimestampCase("1970-01-01 00:00:01", "TIMESTAMP '1970-01-01 00:00:01.000'", id++))
+                .add(new TimestampCase("1970-01-01 00:00:02.1", "TIMESTAMP '1970-01-01 00:00:02.100'", id++))
+                .add(new TimestampCase("1970-01-01 00:00:03.12", "TIMESTAMP '1970-01-01 00:00:03.120'", id++))
+                .add(new TimestampCase("1970-01-01 00:00:04.123", "TIMESTAMP '1970-01-01 00:00:04.123'", id++))
+                .add(new TimestampCase("1970-01-01 00:00:05.1239", "TIMESTAMP '1970-01-01 00:00:05.123'", id++))
+                .add(new TimestampCase("1970-01-01 00:00:06.12399", "TIMESTAMP '1970-01-01 00:00:06.123'", id++))
+                .add(new TimestampCase("1970-01-01 00:00:07.123999", "TIMESTAMP '1970-01-01 00:00:07.123'", id++))
+                .add(new TimestampCase("1970-01-01 00:00:08.1239999", "TIMESTAMP '1970-01-01 00:00:08.123'", id++))
+                .add(new TimestampCase("1970-01-01 00:00:09.12399999", "TIMESTAMP '1970-01-01 00:00:09.123'", id++))
```

Comment:
Slightly changed the values. Otherwise, the subsequent predicates return the duplicated rows.